### ISSUE TITLE
Reads Grepper html code.

### ISSUE THREAD

**Issue by anurag3301** [p00f/cphelper.nvim#9]
The plugin reads the Grepper extension's HTML code and add that to input and output

https://user-images.githubusercontent.com/52702259/127300748-87b5adc7-2286-488e-b5b0-b6d0136fcc78.mp4

**Comment by p00f** [comment]
I think that's a bug in the browser extension, can you link the problem here?

**Comment by p00f** [comment]
Which extension is that (grepper)?

**Comment by p00f** [comment]
In any case, this issue should be filed with the browser extension https://github.com/jmerle/competitive-companion. The plugin simply writes the output of the extension to files

**Comment by p00f** [comment]
If the extension author refuses to fix this (because you are using another extension), I'll try to fix this on my end

**Comment by anurag3301** [comment]
Sure, I think you can just check for that line and remove that completely.
And you can give the user option to turn on or off this feature

**Comment by p00f** [comment]
> I think you can just check for that line and remove that completely.

The output of the extension is still "wrong" though

**Comment by p00f** [comment]
@jmerle ?

**Comment by p00f** [comment]
Until the extension fixes it you can change
https://github.com/p00f/cphelper.nvim/blob/90b63f8a2557c39702b63c240188f12dcfe3ef54/lua/cphelper/prepare.lua#L25-L28 to
```lua
    for i, test in pairs(tests) do
        local input = test["input"]
        input = string.gsub(input, "\n<div class .....>", "")
        local output = test["output"]
        output = string.gsub(output, "\n<div class .....>", "")
        problem_dir:joinpath("input" .. i):write(input, "w")
        problem_dir:joinpath("output" .. i):write(output, "w")
    end
```

**Comment by anurag3301** [comment]
Got it, Thanks, buddy.

**Comment by jmerle** [comment]
I fixed the issue in Competitive Companion 2.25.6 which has just gone live. After it has passed the review stages of the Chrome Web Store and Mozilla's AMO your browser should auto-update the extension within 24 hours.
